test/evmc/mocked_host.hpp

```diff
@@ -18,27 +18,14 @@ using bytes = std::basic_string<uint8_t>;
 /// Extended value (by dirty flag) for account storage.
 struct storage_value
 {
-    /// The storage value.
-    bytes32 value;
+    /// The current storage value.
+    bytes32 current;
 
-    /// True means this value has been modified already by the current transaction.
-    bool dirty{false};
+    /// The original storage value.
+    bytes32 original;
 
     /// Is the storage key cold or warm.
-    evmc_access_status access_status{EVMC_ACCESS_COLD};
-
-    /// Default constructor.
-    storage_value() noexcept = default;
```

Comment:
Why remove these?

Reply:
Not strictly needed if you don't have any other constructor. But I will bring it back to be explicit that solidity is only using this one.